x-pack/solutions/security/plugins/security_solution/server/endpoint/lib/reference_data/saved_objects/mappings.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectsType } from '@kbn/core-saved-objects-server';
+import { SECURITY_SOLUTION_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
+import { REFERENCE_DATA_SAVED_OBJECT_TYPE } from '../constants';
+
+export const referenceDataSavedObjectType: SavedObjectsType = {
+  name: REFERENCE_DATA_SAVED_OBJECT_TYPE,
+  indexPattern: SECURITY_SOLUTION_SAVED_OBJECT_INDEX,
+  namespaceType: 'agnostic',
+  hidden: true,
+  mappings: {
+    dynamic: false,
+    properties: {
+      id: { type: 'keyword' },
+      type: { type: 'keyword' },
+      owner: { type: 'keyword' },
+      metadata: { type: 'object' },
+    },
+  },
+};
```

Comment:
I think that `modelVersions` should be added here. IIUC (please, @gsoldevila keep me honest here), it's not mandatory for the introduction of SOs, but it's beneficial for future upgrades using ZDT (Serverless) algorithm in case we need to rollback a release.

Refer to https://docs.elastic.dev/kibana-dev-docs/tutorials/saved-objects#defining-model-versions for more info.

Reply:
@afharo - adding that now 👍

Reply:
Ok. Done. Can you take a look at the latest